testutil/base.go

```diff
@@ -40,20 +44,23 @@ func Attributes(t testing.TB) []sdk.Attribute {
 
 }
 
+// Resources produces an attribute list for populating a Group's
+// 'Resources' fields.
 func Resources(t testing.TB) []dtypes.Resource {
 	t.Helper()
 	count := rand.Intn(10) + 1
 
 	vals := make([]dtypes.Resource, 0, count)
 	for i := 0; i < count; i++ {
+		coin := sdk.NewCoin(CoinDenom, sdk.NewInt(rand.Int63n(9999)))
```

Comment:
It doesn't really matter much but `Intn(x)` returns something in the range of `[0,x)` (`[0,x-1]`); hence the `Intn(10) + 1` above - I wanted the range `[1,10]`.